Content.Shared/Damage/Systems/DamageOnHighSpeedImpactSystem.cs

```diff
@@ -38,16 +40,17 @@ private void HandleCollide(EntityUid uid, DamageOnHighSpeedImpactComponent compo
         if (speed < component.MinimumSpeed)
             return;
 
+        var speedRatio = speed / component.MinimumSpeed;
```

Comment:
```suggestion
        var speedRatio = Math.Max(speed - component.MinimumSpeed, 0);
```
Damnit "33 hours with no sleep Tyler", you can't just make it multiplicatively scale with speed. What if someone sets the engine speed limit to something absurd like 300 meters per second, and now you get player characters taking a million or so blunt damage? Rewrite this with linear scaling damage.

Reply:
@VMSolidus was this resolved yet?